src/app/views/dashboard/games/manage/game/maturity/maturity.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<div class="row">
 		<div class="col-sm-5 col-sm-push-7">
-			<app-scroll-affix v-if="current" :shouldAffix="Screen.isDesktop" :scrollOffset="15">
+			<app-scroll-affix v-if="current" :disabled="!Screen.isDesktop" :scroll-offset="15">
```

Comment:
Not technically this branch, but do you think we should include the text below the image in the scroll affix? It looks really weird when the image scrolls over the text, and this happens every time.

Reply:
yeah, we can do that